core-web/src/main/java/org/silverpeas/core/web/treemenu/process/TreeBuilder.java

```diff
@@ -194,35 +199,45 @@ private static MenuItem buildFirstLevel(TreeFilter filter, String userId, String
     MenuItem item = new MenuItem("root");
     ArrayList<MenuItem> children = new ArrayList<>();
     item.setChildren(children);
-    // the space
     OrganizationController controller = OrganizationControllerProvider.getOrganisationController();
+
+    // the spaces
     if (filter.acceptNodeType(NodeType.SPACE)) {
       List<SpaceInstLight> rootSpaces =
           controller.getRootSpacesContainingComponent(userId, "kmelia");
+      if (TreeHandler.useOrder)
+        rootSpaces.sort(Comparator.comparing(SpaceInstLight::getOrderNum));
+
```

Comment:
A Java rule: always the body of an if or else-statements is put in brackets, even if there is only a single statement of code

More, this code is duplicated from line 122: it can be defined as a method accepting as parameter the list to sort

Reply:
Done.